src/components/core/Alert.vue

```diff
@@ -27,40 +27,35 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   >
     <template v-slot:actions>
       <v-btn
-        icon
+        :icon="mdiClose"
         @click="closeAlert"
         data-cy="snack-close"
-      >
-        <v-icon>{{ $options.icons.mdiClose }}</v-icon>
-      </v-btn>
+      />
     </template>
-    {{ alert.text }}
+    <p>
+      {{ alert.text }}
+    </p>
+    <p
+      v-if="alert.detail"
```

Comment:
I haven't managed to do anything that caused an alert with "detail" to be raised.

Have you got an example I can try?

Reply:
This triggers it
```diff
diff --git a/src/components/cylc/gscan/GScan.vue b/src/components/cylc/gscan/GScan.vue
index 74cbb144..69bd008f 100644
--- a/src/components/cylc/gscan/GScan.vue
+++ b/src/components/cylc/gscan/GScan.vue
@@ -184,7 +184,7 @@ export default {
 
   methods: {
     scanFilesystem () {
-      mutate({ name: 'scan', args: [] }, {}, this.$workflowService.apolloClient)
+      mutate({ name: 'scran', args: [] }, {}, this.$workflowService.apolloClient)
     },

     /**
```

![image](https://github.com/user-attachments/assets/37ff6564-2767-41dc-9405-0e7db6af12c4)